.github/workflows/build.yml

```diff
@@ -53,7 +53,9 @@ jobs:
       run: go test -v -cover -race -coverprofile=coverage.out ./...
   
     - name: Get Coverage Tool
+      env:
+        GO111MODULE: off
       run: go get github.com/mattn/goveralls
-
+      
     - name: Upload Coverage
-      run: goveralls -coverprofile=coverage.out -service=circle-ci -repotoken=${{ secrets.COVERALLS_TOKEN }}
+      run: goveralls -coverprofile=coverage.out -service=github -repotoken=${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this change is needed? Maybe the service change but not the token change there's a repo secret that holds the coveralls token for the coveralls API.

Reply:
ok ill revert that

Reply:
So it turns out that it can't find ${{ secrets.COVERALLS_TOKEN }} I'll change it back to `-service=circle-ci -repotoken=${{ secrets.GITHUB_TOKEN }}`

Reply:
Lets leave it with coveralls, secrets are restricted to collaborators I'll just give you the collab bit and it will start working.

Reply:
ok so want me to try again with `run: goveralls -coverprofile=coverage.out -service=circle-ci -repotoken=${{ secrets.COVERALLS_TOKEN }}` ? @ianmarmour

Reply:
Yep

Reply:
Still failing because there is no key